src/status_im2/contexts/profile/settings/view.cljs

```diff
@@ -0,0 +1,67 @@
+(ns status-im2.contexts.profile.settings.view
+  (:require [oops.core :as oops]
+            [quo.core :as quo]
+            [quo.theme :as quo.theme]
+            [react-native.core :as rn]
+            [react-native.reanimated :as reanimated]
+            [react-native.safe-area :as safe-area]
+            [status-im2.common.not-implemented :as not-implemented]
+            [status-im2.contexts.profile.settings.constants :as settings.constants]
+            [status-im2.contexts.profile.settings.header.view :as settings.header]
+            [status-im2.contexts.profile.settings.style :as style]
+            [utils.debounce :as debounce]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn- f-settings-item-view
+  [data]
+  [quo/category
+   {:list-type :settings
+    :blur?     true
+    :data      data}])
+
+(defn scroll-handler
+  [event scroll-y]
+  (let [current-y (oops/oget event "nativeEvent.contentOffset.y")]
+    (reanimated/set-shared-value scroll-y current-y)))
+
+(def footer
+  [rn/view {:style style/footer-container}
+   [quo/button {:type :danger :icon-left :i/log-out}
+    (i18n/label :t/logout)]])
+
+(defn- settings-view
+  [theme]
+  (let [insets              (safe-area/get-insets)
+        customization-color (rf/sub [:profile/customization-color])
+        scroll-y            (reanimated/use-shared-value 0)]
+    [quo/overlay {:type :shell}
+     [rn/view {:style {:flex 1}}
+      [rn/view
+       {:style (style/navigation-wrapper {:customization-color customization-color
+                                          :inset               (:top insets)
+                                          :theme               theme})}
+       [quo/page-nav
+        {:background :blur
+         :icon-name  :i/close
+         :on-press   #(rf/dispatch [:navigate-back])
+         :right-side [{:icon-name :i/multi-profile :on-press #(rf/dispatch [:open-modal :sign-in])}
+                      {:icon-name :i/qr-code :on-press not-implemented/alert}
+                      {:icon-name :i/share
+                       :on-press  #(debounce/dispatch-and-chill [:open-modal :share-shell] 1000)}]}]]
+      [rn/flat-list
+       {:header                          [settings.header/view {:scroll-y scroll-y}]
+        :data                            settings.constants/settings-items
+        :key-fn                          str
```

Comment:
Basically, we should never use `str` as the key function for items that are data structures, as this is slow and inefficient. @smohamedjavid and I saw this first hand in the past :) Looks like we can use `:title` as the unique key?